packages/editor/src/components/post-text-editor/index.js

```diff
@@ -65,6 +66,18 @@ export default function PostTextEditor() {
 		}
 	};
 
+	useEffect( () => {
+		valueRef.current = value;
+	}, [ value ] );
+
+	// Ensure changes aren't lost when component unmounts.
+	useEffect( () => {
+		return () => {
+			const blocks = parse( valueRef.current );
```

Comment:
Should there be a check for `isDirty`?

Reply:
I think it's okay to skip dirty check here and assume we always want to sync changes.